app/src/main/java/com/infomaniak/drive/utils/TabViewPagerUtils.kt

```diff
@@ -48,7 +49,7 @@ object TabViewPagerUtils {
             if (isChecked) {
                 val position = (tabsViewPager.adapter as ViewPagerAdapter).tabs.indexOfFirst { it.button == checkedId }
                 tabsViewPager.setCurrentItem(position, true)
-                onCheckedButton?.invoke(position)
```

Comment:
Why did you remove this line?

Reply:
Probably a bad selection on paste, good catch